### ISSUE TITLE
reference-policy should be set to local

### ISSUE THREAD

**Issue by mark-walle** [BCDevOps/pipeline-cli#9]
Pods were failing to image:
```
Failed to pull image "docker-registry.default.svc:5000/8gsiqa-tools/caps@sha256:e66b93cae630d0115fe9ffe39465ae6f2c934554b9c7b9f414337ad93c223c77": rpc error: code = Unknown desc = manifest for docker-registry.default.svc:5000/8gsiqa-tools/caps@sha256:e66b93cae630d0115fe9ffe39465ae6f2c934554b9c7b9f414337ad93c223c77 not found
```

The image in Tools seemed to not be there. I'm still unclear as to why that image was pruned from the Tools namespace. It was tagged.

The image reference was maintained in the local (prod) namespace and so the following command corrected the errors on the pods:
```
oc tag caps@sha256:e66b93cae630d0115fe9ffe39465ae6f2c934554b9c7b9f414337ad93c223c77 caps:prod-1 --reference-policy=local
```

**Comment by mark-walle** [comment]
@cvarjao you mentioned this happened because it had a soft reference to an image in the tools namespace, and that Latest version of the pipeline-cli enforces reference `policy = local` to avoid that (and that this might be in a release branch).

Just wondering if there is an update on this?

**Comment by cvarjao** [comment]
Fix in d6b3c0e8d1383835148a3f50147e7b39f715bf9d
